airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -99,6 +100,12 @@ def get_connection_form_widgets() -> dict[str, Any]:
             "cluster_context": StringField(lazy_gettext("Cluster context"), widget=BS3TextFieldWidget()),
             "disable_verify_ssl": BooleanField(lazy_gettext("Disable SSL")),
             "disable_tcp_keepalive": BooleanField(lazy_gettext("Disable TCP keepalive")),
+            "xcom_sidecar_container_image": StringField(
+                lazy_gettext("XCom sidecar image"), widget=BS3TextFieldWidget()
+            ),
+            "xcom_sidecar_container_resources": StringField(
+                lazy_gettext("XCom sidecar resources (JSON format)"), widget=BS3TextFieldWidget()
+            ),
```

Comment:
I assume to have it working per default, you need to take current hard-coded values as default here? For all who are not setting up explicitly. So that "alpine" is sourced. (Or is the default populated from somewhere else?)

Reply:
Okay, I checked by executing manually because was not aware of the provider - back-end code taking a default was already existing, this PR just adds the glue (again) to expose the config to the connection level.
Works as described, THANKS!